benchmarks/comparison.py

```diff
@@ -9,14 +9,14 @@
 from django.conf import settings
 
 compare_script = os.getenv('COMPARE_SCRIPT', None)
-compare_command = [compare_script, testjob_before.data.file.name, testjob_after.data.file.name]
+compare_command = [compare_script]
 if not compare_script:
```

Comment:
This is always False. Even if compare_script == None than compare_command = [None]. This always resolves as logical True, so negation  will always be False. The if statement is either not needed here or shadows some bug.